### PR TITLE
Matlab code-base

### PR DESCRIPTION
**Pull request by gmrukwa**
This is MATLAB code-base for the project.

As almost none of us understand MATLAB swiftly, only the .gitignore and .NET naming convention should be checked here (in the .prj file of the compiled library) for exported functions and classes.

Compiled version of this project is being uploaded onto One Drive.

### CODE REVIEW

**Comment by gmrukwa** [comment]
[MATLAB code-base](https://trello.com/c/O9bCuqCl/33-matlab-code-base)

File: .gitignore [hunk 1]
```diff
@@ -1,3 +1,12 @@
+## Ignore MATLAB artifacts
+*.asv
+
+## Ignore MATLAB data files
+*.mat
+
+## Ignore MATLAB compiler output
+*-compile/
+
```
Comment (FreexD) [inline .gitignore]: https://www.gitignore.io/api/matlab wouldn't we need some of these as well?

File: MATLAB/algorithms.prj [hunk 2]
```diff
@@ -0,0 +1,211 @@
+<deployment-project plugin="plugin.ezdeploy" plugin-version="1.0">
+  <configuration file="C:\Users\Grzegorz\Source\Repos\MSI-dev\msi-divik\MATLAB\algorithms.prj" location="C:\Users\Grzegorz\Source\Repos\MSI-dev\msi-divik\MATLAB" name="algorithms" target="target.ezdeploy.library" target-name="Library Compiler">
```
Comment (FreexD) [inline MATLAB/algorithms.prj]: The rest of the project members would't be needing that one, right? It's only for compilation on your machine? It contains an absolute path which concerned me a little.

File: .gitignore (hunk 1 above)
Comment (gmrukwa) [inline .gitignore]: We won't compile MEX, nor use SIMULINK. I think those are not necessary. Possibly this one:
```
# OSX / *nix default autosave extension
*.m~
```
may be needed.

File: MATLAB/algorithms.prj (hunk 2 above)
Comment (gmrukwa) [inline MATLAB/algorithms.prj]: This should be generic. I was convinced MATLAB supports any portability by default.

**Comment by FreexD** [comment]
LGTM

**Pull request merged**